### ISSUE TITLE
Wrong scope within arrow functions?

### ISSUE THREAD

**Issue by rfgamaral** [wallabyjs/quokka#111]
### Issue description or question

The issue title and example below are pretty self-explanatory. Shouldn't the output be `Tony Star` just like Google Chrome and Node.js outputs?

### Sample code

```javascript
var name = 'Tony Stark';

var obj = {
    name: 'Steve Rogers',
    printName: () => {
        console.log(this.name);
    }
};

obj.printName();
```

### Quokka.js Console Output

```
​​​​Quokka #1 (node: v7.10.1)​​​​

​​​​​undefined​​​​​ at ​​​this.name​​​ ​quokka.js:6:8​
```

### Chrome/Node.js Console Output

```
​​​​Tony Stark
```

### Code editor version

Visual Studio Code v1.18.0

### OS name and version

Windows

**Comment by rfgamaral** [comment]
That makes sense. The thing was, I was testing that piece code by copy/paste the code into the Node interpreter and not by loading the module directly. In this case, `Tony Stark` is printed. Can I assume that when node launches without passing in a file, everything we write is in the global scope but if we start node with a file, it becomes a module with it's own scope?

**Comment by ArtemGovorov** [comment]
> Can I assume that when node launches without passing in a file, everything we write is in the global scope but if we start node with a file, it becomes a module with it's own scope?

Yes, it's what happens in plain node. Quokka however always starts with a file (even if it's new/unsaved one).

**Comment by rfgamaral** [comment]
Thanks :)
